tests/runc/main.yml

```diff
@@ -0,0 +1,135 @@
+---
+# vim: set ft=ansible:
+#
+# !!!NOTE!!! This playbook was tested using Ansible 2.1; it is recommended
+# that the same version is used.
+#
+# This playbook is a basic functional test for runc.  It configures the
+# host with the required packages and libraries to run the runc upstream
+# tests found at https://fedorapeople.org/cgit/caiqian/public_git/runctst.git
+#
+#  - runc create
+#  - runc delete
+#  - runc exec
+#  - runc init
+#  - runc kill
+#  - runc list
+#  - runc ps
+#  - runc run
+#  - runc start
+#  - runc state
+#  - runc update
+#
+
+- name: runc - Setup
+  hosts: all
+  become: yes
+
+  tags:
+    - setup
+
+  vars_files:
+    - vars.yml
+
+  roles:
+    - role: runc_version_check
+      runc_version: "{{ g_runc_version }}"
+      tags:
+        - runc_version_check
+
+    # Subscribe if the system is RHEL
+    - role: redhat_subscription
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_subscription
+
+    - role: rpm_ostree_install
+      packages: 'gcc python-devel'
+      reboot: false
+      when: ansible_distribution == 'RedHat' or "'CentOS' in ansible_distribution"
+      tags:
+        - rpm_ostree_install
+
+    # Waiting on https://bodhi.fedoraproject.org/updates/FEDORA-2017-6319e78c81
+    #  to fix criu dependency
+
+    - role: rpm_ostree_install
+      packages: 'gcc python-devel redhat-rpm-config criu'
+      reboot: false
+      when: ansible_distribution == 'Fedora'
+      tags:
+        - rpm_ostree_install_fedora
+
+    - role: reboot
+      tags:
+        - reboot
+
+  post_tasks:
+    - name: Turn on user namespaces
+      shell: grubby --args="user_namespace.enable=1" --update-kernel="$(grubby --default-kernel)"
+      when: ("'CentOS' in ansible_distribution" or ansible_distribution == 'Redhat')
+
+    - include: './roles/reboot/tasks/main.yml'
```

Comment:
I'm hitting a failure here when using Ansible 2.1.4

```
TASK [include] *****************************************************************
fatal: [192.168.121.86]: FAILED! => {"failed": true, "reason": "the file_name '/home/miabbott/workspaces/projectatomic/atomic-host-tests/tests/runc/common/ans_reboot.yaml' does not exist, or is not readable"}
```


FWIW, the following warning is thrown at the beginning of the playbook.

```
[DEPRECATION WARNING]: Included file '/home/miabbott/workspaces/projectatomic/atomic-host-tests/tests/runc/./roles/reboot/tasks/../../../common/ans_reboot.yaml' not found, however since this include is not 
explicitly marked as 'static: yes', we will try and include it dynamically later. In the future, this will be an error unless 'static: no' is used on the include task. If you do not want missing includes to be 
considered dynamic, use 'static: yes' on the include or set the global ansible.cfg options to make all inclues static for tasks and/or handlers.
This feature will be removed in a future release. Deprecation 
warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```